device/core/test/_module_client_test.js

```diff
@@ -615,11 +615,8 @@ describe('ModuleClient', function () {
         sinon.spy(fakeTransport, testConfig.enableFunc);
         var client = new ModuleClient(fakeTransport);
 
-        // Calling 'on' twice to make sure it's called only once on the receiver.
```

Comment:
Actually, it's ok if it's called twice on the receiver. The transport should be the one worrying about avoiding duplication of transport operations